app/views/results/index.html.erb

```diff
@@ -1,7 +1,13 @@
 <div class="row my-2">
 <div class="col-lg-12">
   <h3 class="my-2">Showing <%= @results.count %> results</h3>
-  <%= link_to "Export all results as .csv", export_results_path, class: 'btn btn-primary my-2' %>
+
+  <button type="button" class="btn btn-primary my-2" data-toggle="modal" data-target="#exportModal">
+    Export results as .csv
+  </button>
+
+  <%= render "csv_export_modal_form" %>
```

Comment:
I would put the modal at the bottom of the page outside of any divs to keep everything clear

Reply:
Thanks, I will fix this too.